3 reports.sql

```diff
@@ -0,0 +1,20 @@
+--1) I need to know if I have any repeat customers and what they rented.
```

Comment:
Include what the customer rented in the select statement

Reply:
I can't figure out how to include what the customer rented as it is not included in the group by

Reply:
You are right, this question is not in the scope of your knowledge at this point in the course, you can leave it as is.  In a few sessions, you'll learn about something called a cte, which is what you would use in this case.